patch/0001-arm64-default-config-for-sonic-patches.patch

```diff
@@ -0,0 +1,74 @@
+From adf767005a35a2afeb8006c1023633af4de9d78e Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Thu, 17 Oct 2019 18:31:57 +0530
+Subject: [PATCH] arm64 default config for sonic patches
+
+---
+ debian/build/build_arm64_none_arm64/.config | 9 +++++++++
+ 1 file changed, 9 insertions(+)
+
+diff --git a/debian/build/build_arm64_none_arm64/.config b/debian/build/build_arm64_none_arm64/.config
+index 0ce4748..be5b136 100644
+--- a/debian/build/build_arm64_none_arm64/.config
++++ b/debian/build/build_arm64_none_arm64/.config
+@@ -1247,6 +1247,7 @@ CONFIG_NET_ACT_POLICE=m
+ CONFIG_NET_ACT_GACT=m
+ CONFIG_GACT_PROB=y
+ CONFIG_NET_ACT_MIRRED=m
++# CONFIG_NET_ACT_SAMPLE is not set
+ CONFIG_NET_ACT_IPT=m
+ CONFIG_NET_ACT_NAT=m
+ CONFIG_NET_ACT_PEDIT=m
+@@ -1289,6 +1290,7 @@ CONFIG_MPLS_IPTUNNEL=m
+ # CONFIG_HSR is not set
+ # CONFIG_NET_SWITCHDEV is not set
+ CONFIG_NET_L3_MASTER_DEV=y
++# CONFIG_CGROUP_L3MDEV is not set
+ # CONFIG_QRTR is not set
+ # CONFIG_NET_NCSI is not set
+ CONFIG_RPS=y
+@@ -1472,6 +1474,7 @@ CONFIG_NFC_PN533=m
+ CONFIG_NFC_PN533_USB=m
```

Comment:
why you do need this file?

Reply:
Explained the reason in https://github.com/Azure/sonic-linux-kernel/pull/102#discussion_r339895779